src/ext/resources/legacygraphqlapi/schema.graphqls

```diff
@@ -3236,7 +3236,7 @@ type TicketType implements Node {
     Ticket type ID in format `FeedId:TicketTypeId`. Ticket type IDs are usually
     combination of ticket zones where the ticket is valid.
     """
-    fareId: ID!
+    fareId: String
```

Comment:
Making this non-nullable is a breaking change. Can this ever be null?

Reply:
A good notice  - no, I'm not aware of any such case, must have accidentally forgot to add the exclamation mark